README.md

```diff
@@ -55,11 +55,15 @@ Changes to the CDM may be proposed by CDM Working Groups, individual corporate c
 
 **2.2.2 Appointment of CDM Steering Working Group Maintainers:**
```

Comment:
The heading for this section implies it is all about CDM Steering Working Group Maintainers. We're now changing it to cover the core CDM Maintainers. Should we update the section headings in the README to cater for this?